Sources/Sentry/Public/SentryThread.h

```diff
@@ -5,7 +5,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 @class SentryStacktrace;
 
-NS_SWIFT_NAME(Thread)
```

Comment:
`h`: This is a breaking change, and we must wait until the next major to do this.